wire-ios-data-model/Source/ManagedObjectContext/Migration/113-114/OneOnOneConversationMigrationAction.swift

```diff
@@ -97,6 +97,10 @@ final class OneOnOneConversationMigrationAction: CoreDataMigrationAction {
             sameParticipant
         ])
 
+        //  4. sort by their fully qualified conversation ID in ascending oder, and use the first one.
+        // primary_key is basically the qualified id
+        request.sortDescriptors = [NSSortDescriptor(key: "primaryKey", ascending: true)]
```

Comment:
Is the reason why, given two team one on ones, client A might show one and client B might show the other?

Reply:
what do you mean? can you give an example?

Reply:
I understood that part oft he bug is that if there are duplicate one on ones, then it's not always the same one that is shown across different self clients.  I was asking if the cause was because the sorting of duplicated one on one instances was undefined.